### DISCUSSION TITLE
Unable to complete GitHub Student Program verification due to Hungarian student ID format

### DISCUSSION THREAD

**Discussion by zypherift** [community/community#144146] [GitHub Education]
### Hi everyone,

Hi GitHub Team,
I'm reaching out because I'm encountering issues while trying to apply for the GitHub Student Program. I'm a student in Hungary, and I'm unable to complete the verification process due to some limitations with the current system.

The main challenge is that our Hungarian student ID cards are designed differently from what your verification system expects. The required information (my name, valid dates, and school name) is distributed across different sides of the card, but the system only allows me to upload a single image. This is particularly frustrating because this ID is our only official student verification document.

There's also another complication - my school, like all Hungarian educational institutions, has both a full name and an abbreviated name. My student ID shows the official abbreviated name, but I noticed that GitHub's system only recognizes the full name. Also, even though my ID includes English text, I also keep getting told that it doesn't have English text on it, and I should provide a translation.

I would greatly appreciate your guidance on how to proceed with the verification. Is there perhaps an alternative way to verify my student status, or could you help me navigate this situation? I'm very keen to access the Student Program benefits and continue my learning journey on GitHub.

Looking forward to your response and thank you for your time.
Best regards,
zypherift

**Comment by davevad93** [comment]
Hi @zypherift ,

As stated in the [docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student), "**documents that prove your current student status include a picture of your school ID with current enrollment date, class schedule, transcript, and affiliation or enrollment verification letter**". 

If you're experiencing problems with your ID, you may submit any of the above documents with a current date in a `DD/MM/YYYY`, `YYYY-MM-DD`or `MM/DD/YYYY` format.

**Reply by zypherift** [reply]
Just wanted to clarify something - the enrollment date and expiry date are visible on one side of my ID in DD/MM/YYYY format, but my verification is still getting denied.
My class schedule and verification letters are only available through my countries online system. (www.e-kreta.hu)

Should I:
- Take a screenshot of my online class schedule?
- Take a screenshot of my enrollment verification from the university system?
- Write the current date on paper next to my ID?

**Reply by davevad93** [reply]
@zypherift , try one of the first two options and also check out the [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352)

**Comment by ALBERT-NEW-ERA** [comment]
I was also facing the same problem because my school's student ID format was different. However, I was approved after I did this
![image (1) (1)q](https://github.com/user-attachments/assets/240a90af-04ac-45e8-9a9c-a5e8698673ef)
![aaa](https://github.com/user-attachments/assets/ca6adecb-8044-4fd0-9016-8c966286c7c4)

Since the other required information was on the back of my ID, I simply merged the two images. To reduce the file size below 1 MB, I used a website to compress the image.

https://www.resizepixel.com
